src/Panel.php

```diff
@@ -60,7 +60,10 @@ public function getPanel()
 		$installed = $this->decode($lockFile);
 
 		if ($this->error === NULL) {
+			$required = array_filter($required);
+			$installed = array_filter($installed);
```

Comment:
@mpromain Thank you for the PR. Are these `array_filter()` calls needed? If no, I'll remove them.